korge/src/commonMain/kotlin/com/soywiz/korge/view/Views.kt

```diff
@@ -208,9 +208,10 @@ class Views constructor(
 
     var supportTogglingDebug = true
 	var debugViews = false
-	val debugHandlers = FastArrayList<Views.(RenderContext) -> Unit>()
+	var debugFontExtraScale = 1.0
+	val debugHandlers = FastArrayList<Views.(RenderContext, Double) -> Unit>()
 
-    fun addDebugRenderer(block: Views.(RenderContext) -> Unit) {
+    fun addDebugRenderer(block: Views.(RenderContext, Double) -> Unit) {
```

Comment:
Instead of passing it as a primitive as an extra argument, can we put this `extraFontScale` inside the `RenderContext`? Either as a property or as an extrinsic Extra.Property? Maybe with a name like `debugExtraFontScale` or something?

Additionally, are you doing this because we are not properly detecting the device pixel ratio on windows? Could we implement that instead?